client-course-schedulizer/public/dataInterfaces.tsx

```diff
@@ -0,0 +1,60 @@
+enum Semester {
+  Fall = "FA",
+  Interim = "IN", // TODO: Remove?
+  Spring = "SP",
+  Summer = "SU", // TODO: Is this a thing?
+}
+
+enum Half {
+  First = "first",
+  Full = "full",
+  Second = "second",
+}
+
+interface Location {
+  building: string;
+  roomCapacity: number;
+  roomNumber: string;
+}
+
+interface Meeting {
+  // Like "MWThF" (all days on which the given Meeting time and room is applicable)
+  days: string;
```

Comment:
I suggest we make days be an array of Days. It will make things easier to parse.

Reply:
038a906 should address this, good idea